code/modules/chemistry/tools/food_and_drink.dm

```diff
@@ -146,7 +148,11 @@
 		if (istype(W,/obj/item/kitchen/utensil/fork) || istype(W,/obj/item/kitchen/utensil/spoon))
 			if (prob(20) && (istype(W,/obj/item/kitchen/utensil/fork/plastic) || istype(W,/obj/item/kitchen/utensil/spoon/plastic)))
 				var/obj/item/kitchen/utensil/spoon/plastic/S = W
-				S.break_spoon(user)
+				if(istype(S))
+					S.break_spoon(user)
+				var/obj/item/kitchen/utensil/fork/plastic/F = W
+				if(istype(F))
+					F.break_fork(user)
```

Comment:
do you want to return after these? or do you want the message?

Reply:
I do want the message; I thought it was funny for the person to to stare glumly after their fork breaks.